File: 프로그래머스/lv1/12977. 소수 만들기/소수 만들기.py
def solution(nums):
    answer = []
    for i in range(len(nums)):
        for j in range(i+1, len(nums)):
            for k in range(j+1, len(nums)):
                count = 0
                comb = nums[i] + nums[j] + nums[k]
                for n in range(1, comb+1):
                    if comb % n == 0:
                        count += 1
                if count == 2:
                    answer.append(comb)
    return len(answer)

File: 프로그래머스/lv1/12977. 소수 만들기/소수 만들기.js
function solution(nums) {
    const arr = [];
    let answer = 0;
    const isPrimeNumber = (num) => {
        let count = 0;
        for(let i=2; i < parseInt(Math.sqrt(num)) + 1; i++){
            if(num % i === 0) return false;
        }
        return true;
    }
    
    for(let i=0; i < nums.length; i++){
        for(let j=i+1; j < nums.length; j++){
            for(let k=j+1; k < nums.length; k++){
                answer+=(isPrimeNumber(nums[i] + nums[j] + nums[k])) ? 1 : 0;
            }
        }
    }
    
    return answer;
}